### ISSUE TITLE
X3D: Expanding Architectures for Efficient Video Recognition

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#67]
# TL;DR
A study of efficient search for the action recognition networks. The accuracy of each component of the network is improved by sequentially selecting the most efficient factors. The highest accuracy is achieved with 1/5th the number of parameters of the previous study. The networks explored are those with a small number of channels and using high resolution images.

CVPR2020 oral

![Model capacity in GFLOPs (# of multiply-adds x 109)](https://user-images.githubusercontent.com/24524018/92214262-e58b3280-eece-11ea-9b6b-4c452ce3ffd2.png)

# Why it matters:

# Paper URL

https://arxiv.org/abs/2004.04730

# Submission Dates(yyyy/mm/dd)
2020/04/09

# Authors and institutions
Christoph Feichtenhofer
- Facebook AI Research 

# Methods

# Results

# Comments
